scripts/upgrade

```diff
@@ -132,6 +132,16 @@ fi
 # Normalize the URL path syntax
 path_url=$(ynh_normalize_url_path --path_url=$path_url)
 
+#=================================================
+# UPGRADE DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Upgrading dependencies..." --weight=7
+
+ynh_install_app_dependencies $pkg_dependencies
+
+ynh_remove_php
```

Comment:
Just to know, why a remove before the installation ?

Reply:
OK, that's maybe not necessary. But for apps that upgrade from PHP 7.2 to 7.3, etc. over time, that will leave older versions of PHP installed. Isn't that an issue?

Reply:
I took care of that into that PR https://github.com/YunoHost/yunohost/pull/879/files#diff-9d1bacd64942317e00cbf97e9142934cR52-R68

Reply:
Hmm OK for that, I already imported that piece of code. But it's for PHP configurations; what about installed PHP versions that aren't used any more?
Moreover, what code is wrong...?

Reply:
>  Moreover, what code is wrong...?

None ;)

You're right that I didn't take care of removing previous php versions. Not a big deal, but it would be useless to keep it.
The same part of the code should rely to the file `/etc/php/ynh_app_version` to clean useless php version.
Meanwhile, considering those PR not merged yet, we can keep that into Nextcloud script.